src/cases/containers/case-list/case-list.component.spec.ts

```diff
@@ -312,6 +319,42 @@ describe('CaseListComponent', () => {
     });
   });
 
+  describe('isCaseShareVisible()', () => {
```

Comment:
Please add following test for input values.
isCaseShareVisible(true, [])
isCaseShareVisible(false, ['dummy'])
isCaseShareVisible(false, [])